torchvision/transforms/v2/_geometry.py

```diff
@@ -30,6 +35,10 @@
     query_size,
 )
 
+CVCUDA_AVAILABLE = _is_cvcuda_available()
+if CVCUDA_AVAILABLE:
+    cvcuda = _import_cvcuda()
```

Comment:
Is this import needed? I feel we are not using cvcuda module directly anywhere in this file?